src/main/java/org/dice_research/opal/civet/metrics/Dateformat2.java

```diff
@@ -0,0 +1,238 @@
+
+package org.dice_research.opal.civet.metrics;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.RDFNode;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.rdf.model.ResourceFactory;
+import org.apache.jena.rdf.model.SimpleSelector;
+import org.apache.jena.rdf.model.Statement;
+import org.apache.jena.rdf.model.StmtIterator;
+import org.apache.jena.vocabulary.DCAT;
+import org.apache.jena.vocabulary.DCTerms;
+import org.apache.jena.vocabulary.RDF;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.dice_research.opal.civet.Metric;
+import org.dice_research.opal.common.vocabulary.Opal;
+
+/**
+ * The Dateformat metric awards stars based on the the different dateformat in
+ * the dataset. RDF properties dct:issued, dct:modified and distribution is
+ * checked for dates. The more meaningful dates has been awarded with high
+ * ratings and ratings is decremented to low to the least meaningful dates.
+ * Rating criteria: 1."if the date is in the format: DD/MM/YYYY hh:mm:s GMT -
+ * DD/MM/YYYY hh:mm:s GMT then give 5 stars" 2."if the date is in the format:
+ * YYYY.MM.DD hh:mm:s GMT then give 4 stars" 3."if the date is in the format:
+ * YYYY.MM.DD then give 3 stars" 4."if the date is in the format: YYYY then give
+ * 2 stars" 5."if the date property is empty then give 1 star"
+ * 
+ * references: https://www.w3.org/TR/NOTE-datetime
+ * 
+ * @author Aamir Mohammed
+ */
+public class Dateformat2 implements Metric {
+
+	public static int checkDateFormat(String issued) {
+		if (issued.contains("bis") || issued.contains("-") || issued.contains("�")) {
```

Comment:
Remove duplicate branch conditions and change the code like this: 

if (issued.contains("bis")) {
				String[] range = issued.split("bis");
				return checkRange(range);
			} else if (issued.contains("-")) {
				String[] range = issued.split("-");
				return checkRange(range);
			} else if (issued.contains("—")){
				String[] range = issued.split("—");
				return checkRange(range);
			}
		 else {